### ISSUE TITLE
protractor issue: on hold

### ISSUE THREAD

**Issue by crh225** [asadsahi/AspNetCoreSpa#9]
There is a bug with protractor, but it is waiting on another library to be fixed (of course). there is a work around.

https://bugs.chromium.org/p/chromedriver/issues/detail?id=1464#c2

**Comment by crh225** [comment]
see also: https://github.com/AngularClass/angular2-webpack-starter/issues/615

**Comment by asadsahi** [comment]
@crh225  can't see any bug with protractor or webdriver. Works fine for me. I have fixed the e2e with some commands in readme. Run the e2e tests using `npm run e2e` and you should be able succeed with one test verifying the title of the site. Closing this for now.
